src/Features/SupportTesting/Testable.php

```diff
@@ -44,6 +53,11 @@ static function create($name, $params = [], $fromQueryString = [], $cookies = []
         return new static($requestBroker, $initialState);
     }
 
+    /**
+     * @param string $name
```

Comment:
It can be an array, an object or a string.

Reply:
Thanks, fixed